build-system/runner/nailgun/nailgun-runner

```diff
@@ -0,0 +1,1027 @@
+#!/usr/bin/env python
```

Comment:
This may need to go into a `third_party` dir

Reply:
Done.